src/style.css

```
@@ -1 +1,8 @@
-/* styles go here */
+
+body :not(:last-child).form-field {
```

don't increase specificity if it's unnecessary, also it's better to write pseudoclasses after classes
```suggestion
.form-field:not(:last-child) {
```